smartfs.lua

```diff
@@ -99,13 +123,64 @@ function smartfs.add_to_inventory(form,icon,title)
 	end
 end
 
-function smartfs._makeState_(form,player,params,is_inv)
+function smartfs._makeState_(form, newplayer, params, is_inv, nodepos)
+	-- Create object for monitoring of connected players. If no one connected the state can be free'd
+	local _make_players_ = function(form, newplayer)
+		local this = {}
```

Comment:
In lua documentation both is mentioned, an I liked "this" more :/ anyway to be uniform to the rest of the code I changed it already in the "views" commit.